frontend/lib/src/components/shared/BaseColorPicker/BaseColorPicker.tsx

```diff
@@ -108,10 +108,7 @@ class BaseColorPicker extends React.PureComponent<
     const { value } = this.state
     const cursor = disabled ? "not-allowed" : "default"
     const style = { width, cursor }
-    const previewStyle = {
-      cursor,
-      "pointer-events": disabled ? "none" : "auto",
```

Comment:
This was broken and never worked. It needs to be named `pointerEvents`

Reply:
Oh I really wanted this to be fixed!